src/mumble/CMakeLists.txt

```diff
@@ -831,8 +831,8 @@ if(bonjour)
 
 	target_sources(mumble
 		PRIVATE
-			"BonjourClient.cpp"
-			"BonjourClient.h"
+			"Zeroconf.cpp"
+			"Zeroconf.h"
 
 			"${3RDPARTY_DIR}/qqbonjour/BonjourRecord.h"
```

Comment:
imo it is a bit weird to build Bonjour files if an option named `zeroconf` is `ON`...
Maybe we could give this option a more general name? Or add a comment here explaining why we also need the Bonjour stuff?

Reply:
We should probably add a comment.

The reason why the various files (and in turn the classes/structures) have "Bonjour" in the name is due to the third-party API (`qqbonjour`) being written with that library (`dnssd.dll`) in mind.

However, it happens to work perfectly with Avahi because its API matches mDNSResponder's.

Once we start switching to POCO we should probably add support for Avahi/mDNSResponder directly in our `Zeroconf` class.